specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2021-03-15/cosmos-db.json

```diff
@@ -0,0 +1,8590 @@
+          "readOnly": true,
+          "description": "An array that contains of the read locations enabled for the Cosmos DB account.",
+          "items": {
+            "$ref": "#/definitions/Location"
+          }
+        },
+        "locations": {
+          "type": "array",
+          "readOnly": true,
+          "description": "An array that contains all of the locations enabled for the Cosmos DB account.",
+          "items": {
+            "$ref": "#/definitions/Location"
+          }
+        },
+        "failoverPolicies": {
+          "type": "array",
+          "readOnly": true,
+          "description": "An array that contains the regions ordered by their failover priorities.",
+          "items": {
+            "$ref": "#/definitions/FailoverPolicy"
+          }
+        },
+        "virtualNetworkRules": {
+          "type": "array",
+          "description": "List of Virtual Network ACL rules configured for the Cosmos DB account.",
+          "items": {
+            "$ref": "#/definitions/VirtualNetworkRule"
+          }
+        },
+        "privateEndpointConnections": {
+          "type": "array",
+          "readOnly": true,
+          "description": "List of Private Endpoint Connections configured for the Cosmos DB account.",
+          "items": {
+            "$ref": "../../stable/2021-03-15/privateEndpointConnection.json#/definitions/PrivateEndpointConnection"
+          }
+        },
+        "enableMultipleWriteLocations": {
+          "description": "Enables the account to write in multiple locations",
+          "type": "boolean"
+        },
+        "enableCassandraConnector": {
+          "description": "Enables the cassandra connector on the Cosmos DB C* account",
+          "type": "boolean"
+        },
+        "connectorOffer": {
+          "description": "The cassandra connector offer type for the Cosmos DB database C* account.",
+          "$ref": "#/definitions/ConnectorOffer"
+        },
+        "disableKeyBasedMetadataWriteAccess": {
+          "description": "Disable write operations on metadata resources (databases, containers, throughput) via account keys",
+          "type": "boolean"
+        },
+        "keyVaultKeyUri": {
+          "description": "The URI of the key vault",
+          "type": "string"
+        },
+        "defaultIdentity": {
+          "description": "The default identity for accessing key vault",
+          "type": "string"
+        },
+        "publicNetworkAccess": {
+          "description": "Whether requests from Public Network are allowed",
+          "$ref": "#/definitions/PublicNetworkAccess"
+        },
+        "enableFreeTier": {
+          "description": "Flag to indicate whether Free Tier is enabled.",
+          "type": "boolean"
+        },
+        "apiProperties": {
+          "description": "API specific properties.",
+          "type": "object",
+          "$ref": "#/definitions/ApiProperties"
+        },
+        "enableAnalyticalStorage": {
```

Comment:
2021-03-01-preview introduced new properties around restorable DBs that this API version does not have.  Is it intended to have a newer stable version that does not bring those forward?

Reply:
yes this is intended